### ISSUE TITLE
Text editor does not work in Pipeline replay

### ISSUE THREAD

**Issue by YLPeng** [jenkins-contrib-themes/jenkins-neo-theme#36]
When using Pipeline replay feature, the text editor is buggy as the cursor is shifted a few places from actual typing position.

**Comment by TobiX** [comment]
This is fixed in my Neo2 theme: https://tobix.github.io/jenkins-neo2-theme/

**Comment by lifeofguenter** [comment]
@TobiX can we get that fix in here as well?

**Comment by TobiX** [comment]
I don't think so, since nobody is maintaining the neo theme. That's the reason I made the Neo2 theme. I forked my own theme when nobody merged #27 ...
